src/components/UserInfo.js

```diff
@@ -12,8 +13,13 @@ export default class UserInfo {
   }
 
   setUserInfo(data) {
-    this._displayName.textContent = data.title;
-    this._displayJob.textContent = data.description;
+    this._displayName.textContent = data.name;
+    this._displayJob.textContent = data.about;
+    this._displayImage.src = data.avatar;
```

Comment:
`    this._displayImage.src = data.avatar;
`

Please, use method `setUserImage` here not to duplicate the same code

Reply:
Completed!